### PR TITLE
MINOR: Document change to mention  hadoop/Kerberos Plugins moved to package org.apache.solr.security.hadoop

### PR DESCRIPTION
**Pull request by senthh**
https://issues.apache.org/jira/browse/SOLR-16826

# Description

Document change to mention  hadoop/Kerberos Plugins moved to package org.apache.solr.security.hadoop
# Solution

Added document change in 'major-changes-in-solr-9.adoc file

# Tests

NA
# Checklist

Please review the following and check all that apply:

- [x] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [x] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [x] I have developed this patch against the `main` branch.
- [x] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [x] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

File: solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc [hunk 1]
```diff
@@ -175,7 +175,7 @@ Administrators that need Solr exposed more broadly can change the `SOLR_JETTY_HO
 If you rely upon the previous behavior, then you can change the `clientPortAddress` in `solr/server/solr/zoo.cfg`
 * Jetty low level request-logging in NCSA format is now enabled by default, with a retention of 3 days worth of logs.
 This may require some more disk space for logs than was the case in version 8.x. See xref:deployment-guide:configuring-logging.adoc[Configuring Logging] for how to change this.
-* Hadoop authentication support has been moved to the new `hadoop-auth` module. Existing Solr configurations do not need any Hadoop authentication related changes, however the module needs to be installed - see the section xref:deployment-guide:hadoop-authentication-plugin.adoc[].
+* Hadoop authentication support has been moved to the new `hadoop-auth` module. Users need to add the module hadoop-auth to classpath. The plugins has also changed package name to org.apache.solr.security.hadoop, but can still be loaded as shortform class="solr.HadoopAuthPlugin" or class="solr.ConfigurableInternodeAuthHadoopPlugin" or class="solr.KerberosPlugin"  - see the section xref:deployment-guide:hadoop-authentication-plugin.adoc[].
```
Comment (janhoy) [inline solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc:178]: Code formatting:
```suggestion
* Hadoop authentication support has been moved to the new `hadoop-auth` module. Users need to add the module `hadoop-auth` to classpath. The plugins has also changed package name to `org.apache.solr.security.hadoop`, but can still be loaded as shortform `class="solr.HadoopAuthPlugin"`, `class="solr.ConfigurableInternodeAuthHadoopPlugin"` or `class="solr.KerberosPlugin"`  - see the section xref:deployment-guide:hadoop-authentication-plugin.adoc[].
```

Comment (senthh) [reply solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc]: Thanks for the review @janhoy , I have modified as per your suggestion

**Review by janhoy — APPROVED** [review]

**Comment by HoustonPutman** [comment]
This looks great to me. I can take care of backporting to all of the branches @janhoy , so that all versions of the ref guide get this fix.

**Comment by HoustonPutman** [comment]
Thanks @janhoy !

**Comment by janhoy** [comment]
I merged to main, `9x`, `9_2`, `9_1` and `9_0`, think that are all the guide versions?

**Comment by risdenk** [comment]
Thanks @janhoy and @senthh - SOLR-13989 is what introduced this. When I looked through our docs nothing specifically had the full class always `solr.ABC` so it didn't occur to me to put this in the upgrade notes. Sorry.

**Pull request merged**